sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ServersTable.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useContext, useEffect } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  DataFilters,
+  Toolbar,
+  DataSearch,
+  Data,
+  ResponsiveContext,
+  DataTable,
+  Pagination,
+} from 'grommet';
+import {
+  StatusCriticalSmall,
+  FormUpload,
```

Comment:
We don't want to use "Form" icons anymore because they render smaller than our standard ones, can you switch to just "Upload" instead?

Reply:
changed